src/protocolsupport/protocol/packet/middleimpl/serverbound/play/v_pe/PositionLook.java

```diff
@@ -51,7 +52,10 @@ public void readFromClientData(ByteBuf clientdata) {
 		RecyclableArrayList<ServerBoundPacketData> packets = RecyclableArrayList.create();
 		MovementCache movecache = cache.getMovementCache();
 		NetworkEntity player = cache.getWatchedEntityCache().getSelfPlayer();
-		//movecache.updatePEPositionLeniency(y);
+		double yOffset = (y - movecache.getPEClientY()) > 0.01 ? 0.6 : 0;
```

Comment:
ah... magic numbers im afraid to touch at this point

Reply:
Fine for now though 😄